style.css

```diff
@@ -356,59 +367,1108 @@ span {
+  margin-top: 0.5rem;
+  border: none;
+}
+
+.form-btn:hover {
+  background: #fff;
+  color: #1a2236;
+  border: 1px solid #1a2236;
+}
+
+.form-btn :checked {
+  background: #fff;
+  border: 2px solid #1a2236;
+}
+
+.get {
+  color: #98dede;
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 0.7rem;
+  margin-right: 0.5rem;
+}
+
+.get > li:nth-child(1) {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.get > li:nth-child(2) {
+  margin-top: 1rem;
+  font-size: 2rem;
+}
+
+.contact-footer {
+  display: flex;
+  flex-direction: column;
+  background: #1a2236;
+  width: 100%;
+  height: 150px;
 }
 
-@media screen and (min-width: 768px) {
+.contact-social {
+  display: flex;
+  flex-direction: row;
+  list-style: none;
+  gap: 1rem;
+  justify-content: space-around;
+  margin-right: 2rem;
+}
+
+.project-card-d1,
+.project-d-header,
+.img1,
+.img2,
+.headline-p-d {
+  display: none;
+}
+
+.contact-logo1-d,
+.contact-logo2-d {
+  display: none;
+}
+
+@media screen and (min-width: 500px) and (max-width: 767px) {
+  .headline-p-d {
+    display: none;
+  }
+
   .headline-p {
     width: 100%;
-    height: 10%;
+  }
+
+  .project-cards {
+    display: none;
+  }
+
+  .project-d-header {
+    display: none;
+  }
+
+  .projects-d-container {
+    display: none;
+  }
+
+  .headline-img {
+    background-size: 100% 100%;
   }
 
   .headline-btn,
+  .form-btn,
   .aboutme-btn {
     width: 95%;
   }
 
+  .aboutme-techs,
+  .aboutme-framew,
+  .aboutme-skills {
+    width: 100%;
+    margin-right: 0.5rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .aboutme-cards {
+    padding-right: 2.5rem;
+  }
+
+  .aboutme-p,
+  .contact-para {
+    width: 100%;
+  }
+
+  .logo-corner1,
+  .logo-corner,
+  .logo-corner2 {
+    left: calc(77.5% + 2rem);
+  }
+
+  .contact-form {
+    width: 100%;
+  }
+
+  .contact-footer {
+    width: 100%;
+  }
+
+  .contact input[type="text"] {
+    width: 100%;
+  }
+
+  .techs,
+  .framew {
+    padding-right: 10%;
+  }
+
+  .project-card-d1,
+  .project-d-header,
+  .img1,
+  .img2,
+  .headline-p-d {
+    display: none;
+  }
+
+  .contact1-img,
+  .contact2-img {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 769px) {
+  .header-m > .logo,
+  .menu {
+    display: none;
+  }
+
+  .header-m {
+    justify-content: center;
+  }
+
+  .menu-d-list {
+    list-style: none;
+    display: flex;
+    gap: 1rem;
+  }
+
+  .menu-d-list > li {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+  }
+
+  .headline {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .headline-d {
+    display: flex;
+    flex-direction: column;
+    padding: 6% 0 15% 0;
+    gap: 1rem;
+    align-content: flex-start;
+    margin: 0 0 0 3rem;
+  }
+
+  .myname {
+    font-size: 80px;
+  }
```

Comment:
- [ ] Please use the styles detailed in the Figma design like font-size, font-weight, font-family, color, etc to be similar to the original design 👇🏽 

| Figma design | Your design |
|---|---|
| ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/f9c485ef-c5ec-4a23-8d8c-4030e770e25c) | ![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/58642949/7762502a-e439-4dfe-8c5f-d388bef8a508) |

Reply:
Hi  thanks for your review but pleased i used it please check the work at  1440px i worked on different screens so i modified the font size on smaller screen.Please check .